### PR TITLE
Support examples for role entrypoints

### PR DESCRIPTION
**Pull request by felixfontein**
Ref: https://github.com/ansible/ansible/pull/82671
Ref: https://github.com/ansible/ansible/issues/81734

### CODE REVIEW

**Review by ssbarnea — APPROVED** [review]

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Blocked until core changes are merged.

**Comment by felixfontein** [comment]
Marking as ready for review since https://github.com/ansible/ansible/pull/82671 has been merged.

**Review by ssbarnea — APPROVED** [review]

**Comment by felixfontein** [comment]
@ssbarnea @audgirka thanks!

**Pull request merged**
